### PR TITLE
[MRG] Final Edits Joss

### PR DESCRIPTION
**Pull request by alexrockhill**
Fixes https://github.com/mne-tools/mne-python/pull/10327.

### CODE REVIEW

**Comment by danielskatz** [comment]
Can you also add the comma I suggested in line 51 in [my PR]( #10327)?

**Comment by alexrockhill** [comment]
Both suggestions on line 51 are for commas after the second item and before the `and` in lists of three items. I don't use Oxford commas in my publications so I would prefer not to make this change to keep my work consistent. If you insist that this change is necessary for clarity, I will go ahead and change it.

File: paper.md [hunk 1]
```diff
@@ -78,9 +78,9 @@ These intracranial location and analysis steps are shown in the MNE-Python tutor
...
...ol various aspects of the visualization (e.g., electrode contact marker size, opacity, multiple views) makes this task much easier, both for sEEG and ECoG. Additionally, compared to this previous work, MNE-Python has added key features. First it utilizes an SDR morph to map electrode contact locations onto a template brain, which is orders of magnitude faster (approximately 15 minutes compared to 15 hours) and is comparably accurate [@AvantsEtAl2008]. Second, it has a "snap to center" of the electrode contact feature to both increase accuracy and repeatability. This feature uses the center of mass of all voxels that monotonically decrease from the highest intensity voxel nearby the selected location. Next, it has an integrated 3D view which updates upon selection. Finally, it uses a predetermined list of channel names from the recording file to speed up the localization process and to eliminate matching errors. In general, the intracranial electrode location and analysis in MNE-Python is implemented using coding best-practices, it is feature complete for intracranial analysis, and it has user-friendly features. Importantly, it is poised to be a well-maintained tool into the future.
```
Comment (agramfort) [inline paper.md:83]: @larsoner @drammock I suspect you could add some grant acknowlegments just below. The MNE RO1 and the CZI grants I guess. @alexrockhill we still have time to add this?

Comment (alexrockhill) [reply paper.md:83]: Yes, I think it should be published very soon but if you add them now, it will be in time.

Comment (larsoner) [reply paper.md:83]: Part in https://github.com/mne-tools/mne-python/pull/10332

For CZI perhaps we should add EOSS2 to cover https://chanzuckerberg.com/eoss/proposals/improving-usability-of-core-neuroscience-analysis-tools-with-mne-python/ , @alexrockhill can you add this line for @drammock ?

**Pull request merged**